ivy_tests/test_ivy/test_functional/test_experimental/test_nn/test_losses.py

```diff
@@ -55,6 +55,54 @@ def test_huber_loss(
     )
 
 
+# kl_div
+@handle_test(
+    fn_tree="functional.ivy.experimental.kl_div",
+    dtype_and_input=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
When using the decorator for supported / unsupported dtypes, always make sure to use valid dtypes during testing

```suggestion
        available_dtypes=helpers.get_dtypes("valid"),
```